mmdet3d/models/roi_heads/roi_extractors/single_roipoint_extractor.py

```diff
@@ -0,0 +1,63 @@
+import torch
+from torch import nn as nn
+
+from mmdet3d import ops
+from mmdet3d.core.bbox.structures import rotation_3d_in_axis
+from mmdet.models.builder import ROI_EXTRACTORS
+
+
+@ROI_EXTRACTORS.register_module()
+class Single3DRoIPointExtractor(nn.Module):
```

Comment:
Can we inherits those in MMDet

Reply:
BaseROIExtractor need some args we don't use such as `out_channels`, `featmap_strides`. So I think it  is unnecessary